01-intro.Rmd

```diff
@@ -64,16 +64,16 @@ For certain grant mechanisms for projects that do not generate data, compliance
 The following outline several key questions:
 
 - [**Is my research exempt from the policy?**](#no-sharing)
-- [**Does my research generate scientific data?**](#scientific-data) 
+- [**Does my research generate scientific data?**](#scientific-data)
+- [**Do grant renewals require compliance with the policy?**](#grant-renewals)
+- [**How will the policy impact the review process?**](#review-process)
 - [**When do I need to share my data?**](#sharing-timeline) 
 - [**When should I not share data?**](#when-to-not-share) 
 
 To determine if your research requires compliance with other policies that may influence how you share your data, take this [quiz](https://sharing.nih.gov/other-sharing-policies/which-policies-apply-to-my-research).
 
 
-## During grant applications 
-
```

Comment:
I was trying to make it easier for people to find which grant mechanisms must comply from the TOC